src/Text/Pandoc/Writers/Markdown.hs

```diff
@@ -600,6 +600,8 @@ blockToMarkdown' opts (OrderedList (start,sty,delim) items) = do
 blockToMarkdown' opts (DefinitionList items) = do
   contents <- inList $ mapM (definitionListItemToMarkdown opts) items
   return $ mconcat contents <> blankline
+blockToMarkdown' opts (Figure (ident, classes, kvs) _ body) =
+  blockToMarkdown' opts (Div (ident, ["figure"] `union` classes, kvs) body)
```

Comment:
Here I added the `"figure"` class to the `div`. I'd like to know how this aligns with the goal of keeping English words out of markdown.